io/net/bonding.py

```diff
@@ -221,9 +253,13 @@ def bond_setup(self, arg1, arg2):
                     self.fail("unable to interface up")
             cmd = "ifconfig %s %s netmask %s up"\
                   % (self.bond_name, self.host_ips[0], self.net_mask[0])
-            if process.system(cmd, shell=True, ignore_status=True) != 0:
-                self.fail("bond setup command failed in local machine")
-            time.sleep(5)
+            for i in range(0, 600, 60):
+                if process.system(cmd, shell=True, ignore_status=True) != 0:
```

Comment:
Not needed to run the command every minute. This should have been before the `for` loop